project-base/webpack.config.js

```diff
@@ -7,6 +7,7 @@ const yaml = require('js-yaml');
 const fs = require('fs');
 const path = require('path');
 const StylelintPlugin = require('stylelint-webpack-plugin');
+const sources = require('./assets/js/bin/helpers/sources');
```

Comment:
This constant seems unused.

Reply:
This is broken rebase e68b13b88a9abc1d30a52ca8965850125b01f045